code/global_vr.dm

```diff
@@ -35,4 +35,6 @@ var/global/list/acceptable_fruit_types= list(
 											"tomato",
 											"watermelon",
 											"wheat",
-											"whitebeet")
\ No newline at end of file
+											"whitebeet")
+
+mouse_respawn_time = 2.5 // Amount of time that must pass between a player dying as a mouse and repawning as a mouse. In minutes. VOREStation Edit - Changed timer to 2.5, halved from 5.
```

Comment:
Oi put this at the top of the file, not the bottom. Also will this even compile? There's no object path.

Reply:
I'll get that done! Ascian said to remove the var/ - if Travis fails again after moving it I'll try the var/

Reply:
I'm getting this when moving that line to the top of the file.

`code\global.dm:56:error: mouse_respawn_time: duplicate definition`
`code\global_vr.dm:1:error: mouse_respawn_time: previous definition`

Granted, I tried var/mouse_respawn_time = - I'm going to remove the var/ and see if it successfully compiles again.

Reply:
Okay, with var/ removed, the error changes to this:

code\global_vr.dm:1:error: mouse_respawn_time: duplicate definition
code\global_vr.dm:1:error: 2.5: duplicate definition
code\global_vr.dm:1:error: =: instruction not allowed here

Should I be commenting out the original var/mouse_respawn_time in the Polaris files and using var/mouse_respawn_time in the _vr.dm?

Reply:
I guess you might as well just go back to your original vorestation edit. Oh well.